openmetadata-ui/src/main/resources/ui/src/components/DashboardDetails/DashboardDetails.component.tsx

```diff
@@ -332,6 +334,42 @@ const DashboardDetails = ({
       tagUpdateHandler(updatedDashboard);
     }
   };
+
+  const handleRestoreTable = () => {
+    const restoreDashboardData = omit(dashboardDetails, [
+      'id',
+      'fullyQualifiedName',
+      'version',
+      'updatedAt',
+      'updatedBy',
+      'href',
+      'serviceType',
+      'followers',
+      'usageSummary',
+      'changeDescription',
+      'deleted',
+    ]);
```

Comment:
Instead of doing this can we create a new object based on the schema, so that way we don't miss any important field